packages/cli/src/storybook/getSbOption.ts

```diff
@@ -0,0 +1,12 @@
+export function getSbOption<T>(key: string, fallback?: T) {
```

Comment:
one problem i see here is that this is now bound to react, i separated the core logic from react specific storybook stuff previously. i think we should add a storybook part to the config step where init all this shit and then in the decorator part instead of importing it directly we use

`config.storybook.options`

then we have a better separation of concerns and compose all the storybook stuff in one place